akka-sample-cluster-client-grpc-scala/README.md

```diff
@@ -1,7 +1,7 @@
 # Cluster Client with gRPC transport
 	
 See purpose of this example and important clarifications of when to use this approach in
-[Migration to Akka gRPC](https://doc.akka.io/docs/akka/2.6/cluster-client.html#migration-to-akka-grpc).
+[Migration to Akka gRPC](https://pekko.apache.org/docs/pekko/current/cluster-client.html#migration-to-akka-grpc).
```

Comment:
should be https://pekko.apache.org/docs/pekko/current/cluster-client.html#migration-to-apache-pekko-grpc

Would be a good idea to try the URLs to see what works

Reply:
updated